### DISCUSSION TITLE
What to Choose Between Admin Panel and separate packages of Form Builder, Table and other

### DISCUSSION THREAD

**Discussion by imrancoder** [filamentphp/filament#5641] [General]
If we Consider their is a Project Like ERP where Too Many Models(200+) and Multi Context, then in that case should we use Admin Panel or we should manage it own by using Form Builder and Table separately with Laravel Breeze. 

in many places needs to build custom views for layout and navigations

so managing files structure will be more complex on Admin Panel or not ?

what will be the suggestions of @danharrin and other community members if you build that type of Application.

**Comment by ThijmenKort** [comment]
In my opionion to keep everything in your own control you shoud just use the packages seperately. I only suggest using the Admin Panel if you don't have to override anything about the layout of the panel.

**Reply by danharrin** [reply]
Agreed

**Comment by imrancoder** [comment]
Thanks
